### DISCUSSION TITLE
Help: Using XPRA, I can not see colored cells in Libreoffice

### DISCUSSION THREAD

**Discussion by gold-island-pirate** [Xpra-org/xpra#4321] [General]
Hi,

I'm using XPRA in Linut Mint and everything is good, including sound, except that I can not see colored cells in Libreoffice.
Can somebody help?

**Comment by totaam** [comment]
Please see https://github.com/Xpra-org/xpra/wiki/Reporting-Bugs

**Reply by gold-island-pirate** [reply]
Hi,

Ok, I've seen https://github.com/Xpra-org/xpra/wiki/Reporting-Bugs

Please, see the info in the 4 attached text files:

[details.txt](https://github.com/user-attachments/files/16629742/details.txt)
[xpra info - server.txt](https://github.com/user-attachments/files/16629743/xpra.info.-.server.txt)
[xpra showconfig - client.txt](https://github.com/user-attachments/files/16629744/xpra.showconfig.-.client.txt)
[xpra showconfig - server.txt](https://github.com/user-attachments/files/16629745/xpra.showconfig.-.server.txt)

**Reply by totaam** [reply]
Extracting the useful information added as text files...
> `Xpra GTK3 X11 client version 3.1.3 64-bit`

Here you go. There is your answer:
https://github.com/Xpra-org/xpra/wiki/Distribution-Packages

Don't use broken xpra packages.
